cli/tests/integrations/test_package.py

```diff
@@ -401,6 +401,25 @@ def test_uninstall_cli_only_when_no_apps_remain():
         assert str(exc_info.value) == "'helloworld' is not a dcos command."
 
 
+def test_uninstall_cli_only_implicitly_does_not_prompt_confirmation():
+        stdout = (
+            b'By Deploying, you agree to the Terms '
+            b'and Conditions https://mesosphere.com/'
+            b'catalog-terms-conditions/#community-services\n'
+            b'Installing CLI subcommand for package [helloworld] '
+            b'version [0.1.0]\n'
+            b'New command available: dcos ' +
+            _executable_name(b'helloworld') +
+            b'\n'
+        )
+
+        with _package(name='helloworld',
+                      args=['--yes', '--cli'],
```

Comment:
I'm confused by --cli being present in a test for "implicitly".

Reply:
This flag is for installation command only, the implicitness is about uninstallation (where we don't specify `--cli`).